CMSgov/design-system
# [WNMGDS-325] Fix position sticky in IE11

### Summary
`<HelpDrawer>` uses `position: sticky` to position the help drawer title, this style is not supported in IE11. 

### How to test
1. Run the branch locally
2. Edit `HelpDrawer.example.jsx` to use a very long title on line 47
3. Install and use [ngrok](https://ngrok.com/) or [localtunnel](https://localtunnel.github.io/www) in order to expose your local branch to a public URL
4. Setup live testing on IE11 on SauceLabs or your own VM to test. 
5. Ensure the long title does not overlap with the help drawer content
